docs/release_notes/7.0.18-11.md

```diff
@@ -0,0 +1,78 @@
+* [SERVER-76883](https://jira.mongodb.org/browse/SERVER-76883) - Improved logging behavior for roles not being mapped to users to throw a warning for internally-authorized users and to show this message with higher log verbosity level for externally-authorized users so that this message can be suppressed
+* [SERVER-85627](https://jira.mongodb.org/browse/SERVER-85627) - Extended logs for a thread is interrupted by replica set stage change with details like thread name and the reason for its interruption 
+* [SERVER-89614](https://jira.mongodb.org/browse/SERVER-89614) - Added enumeration to track the cleanup state for `$out` stage to properly drop buckets collection during interrupt 
+* [SERVER-95452](https://jira.mongodb.org/browse/SERVER-95452) - Prevented CollectionRoutingInfoTargeter from iterating all chunk ranges during update and delete to improve performance
+* [SERVER-98720](https://jira.mongodb.org/browse/SERVER-98720) - Added redaction of the BSON command for "Plan executor error" warning logs to prevent the entire command text (possibly including PII) to end up in the logs
+* [SERVER-100901](https://jira.mongodb.org/browse/SERVER-100901) - Reduce the strictness of an invariant check for the unique ID (uid) in the session identifier (lsid) to match between the operation's context (opCtx) and the incoming command request by treating it as a code error, logging it and have the server continue operating
+
+Find the full list of changes in the [MongoDB 7.0.17 Community Edition release notes](https://www.mongodb.com/docs/manual/release-notes/7.0/#7.0.17---feb-21--2025) and [MongoDB 7.0.18 Community Edition release notes](https://www.mongodb.com/docs/manual/release-notes/7.0/#7.0.18---mar-19--2025).
+
+## Changelog
+
+### New Features
+
+* [PSMDB-1282](https://perconadev.atlassian.net/browse/PSMDB-1282) - Add the `security.kmip.useLegacyProtocol` config file option to simplify migration from MongoDB Enterprise Edition to Percona Server for MongoDB
+
+### Improvements
+
+* [PSMDB-1620](https://perconadev.atlassian.net/browse/PSMDB-1620) - Do not create an empty audit log file if the log destination is not `file`
+
+* [PSMDB-1621](https://perconadev.atlassian.net/browse/PSMDB-1621) - Check if an audit log file can be opened for writing at default path
+
+
+### Fixed Bugs
+
+* [PSMDB-118](https://perconadev.atlassian.net/browse/PSMDB-118) - Fixed the audit log file extension to be created based on the specified audit log format.
+
+* [PSMDB-121](https://perconadev.atlassian.net/browse/PSMDB-121) - Fixed the issue with the server failing to create an audit log file at the user provided path when the process is forked and the relative path is given by converting it to an absolute path
+
+* [PSMDB-1227](https://perconadev.atlassian.net/browse/PSMDB-1227) - Allowed  setting a Distinguished Name(DN) value in attributes part of the LDAP query
```

Comment:
This description is ambiguous and may lead to the wrong conclusion that distinguished name value like "cn=some,dn=thing,d=com" may appear i the attributes part of the LDAP query. In fact this ticket was about server's behaviour when attributes part equals the "dn" string.
Consider following description:
> Now the "dn" string in the attributes part of LDAP query is treated in the same way as empty string to be more compatible with commonly used configs.